detox/src/devices/common/drivers/android/emulator/exec/EmulatorExec.js

```diff
@@ -74,7 +74,7 @@ class LaunchCommand extends ExecCommand {
     if (this._options.headless) {
       switch (os.platform()) {
         case 'darwin':
-          return 'host';
+          return 'off';
```

Comment:
@d4vidi just to make sure everyone's okay with it.

Reply:
Igor thinks it is an error on our side to have `host` default. Maybe it is, maybe not, let's think.

Reply:
This isn't good enough because it would affect all detox users out there (running macs). We should apply `--gpu` explicitly as a command-line argument in the scripts, or conditionally (local vs. CI) in the Detox configuration scripts.